### PR TITLE
metrics_per_epoch 2

### PR DESCRIPTION
**Pull request by shijiegu**
Now everything is working. Please be careful now to change the code too much again.

### CODE REVIEW

File: franklab_mountainsort/core.py [hunk 1]
```diff
@@ -2,13 +2,18 @@
...
+import functools
+import multiprocessing
 
 import franklab_mountainsort.ms4_franklab_pyplines as pyp
 import numpy as np
 import pandas as pd
+import pathlib
 
+METRICS_INPUT = 'metrics_merged.json'
+METRICS_OUTPUT = 'metrics_merged_tagged.json'
```
Comment (edeno) [inline franklab_mountainsort/core.py]: Delete

File: franklab_mountainsort/ms4_franklab_proc2py.py [hunk 2]
```diff
@@ -307,48 +306,8 @@ def pyms_merge_burst_parents(firings, metrics, firings_out, opts=None):
     )
 
 
-def tagged_curation(cluster_metrics, metrics_tagged,
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_proc2py.py:310]: Add this back

File: franklab_mountainsort/ms4_franklab_proc2py.py [hunk 3]
```diff
@@ -307,48 +306,8 @@ def pyms_merge_burst_parents(firings, metrics, firings_out, opts=None):
...
-            'isolation_thresh': isolation_thresh,
-            'noise_overlap_thresh': noise_overlap_thresh,
-            'peak_snr_thresh': peak_snr_thresh,
-            'mv2file': mv2file,
-        },
-        opts
-    )
-
 
-def get_mda_list(animal, date, ntrode, data_location):
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_proc2py.py:351]: Add this back

File: franklab_mountainsort/ms4_franklab_pyplines.py [hunk 4]
```diff
@@ -337,23 +348,23 @@ def merge_burst_parents(dataset_dir, output_dir):
         samplerate=ds_params['samplerate'])
 
 
-def add_curation_tags(dataset_dir, output_dir, firing_rate_thresh=0.01,
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_pyplines.py:340]: Change this back.

File: franklab_mountainsort/ms4_franklab_pyplines.py [hunk 5]
```diff
@@ -363,60 +374,195 @@ def add_curation_tags(dataset_dir, output_dir, firing_rate_thresh=0.01,
 
     '''
     pyms_add_curation_tags(
-        metrics=os.path.join(dataset_dir, metrics_input),
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_pyplines.py]: Change this back.

File: franklab_mountainsort/ms4_franklab_pyplines.py [hunk 6]
```diff
@@ -363,60 +374,195 @@ def add_curation_tags(dataset_dir, output_dir, firing_rate_thresh=0.01,
...
-        metrics_tagged=os.path.join(output_dir, metrics_output),
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_pyplines.py]: Change this back.

**Review by edeno — APPROVED** [review]

File: franklab_mountainsort/ms4_franklab_proc2py.py [hunk 7]
```diff
@@ -348,7 +344,8 @@ def tagged_curation(cluster_metrics, metrics_tagged,
     )
 
 
-def get_mda_list(animal, date, ntrode, data_location):
+
+def get_mda_list(anim, date, ntrode, data_location):
```
Comment (edeno) [inline franklab_mountainsort/ms4_franklab_proc2py.py]: `anim` should be `animal`

File: franklab_mountainsort/core.py (hunk 1 above)
Comment (edeno) [reply franklab_mountainsort/core.py]: Still need to delete.

**Review by edeno — CHANGES_REQUESTED** [review]

**Comment by shijiegu** [comment]
Hey Eric! I just updated!

**Review by edeno — APPROVED** [review]

**Pull request merged**